### ISSUE TITLE
bug: `Found a 'popover' attribute with an invalid value.

### ISSUE THREAD

**Issue by slwhitman** [ionic-team/ionic-framework#27584]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v5.x

### Current Behavior

Using Chromium Build 114.0.5735.90 or 114.0.5735.91 When activating a popover item, the popover dropdown menu fails and the console logs the following:

```Found a 'popover' attribute with an invalid value.
(anonymous) @ framework-delegate-d1eb6504.js:15
d @ framework-delegate-d1eb6504.js:2
(anonymous) @ framework-delegate-d1eb6504.js:2
(anonymous) @ framework-delegate-d1eb6504.js:2
n @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ framework-delegate-d1eb6504.js:1
(anonymous) @ ion-popover-ios.entry.js:310
l @ ion-popover-ios.entry.js:2
(anonymous) @ ion-popover-ios.entry.js:2
(anonymous) @ ion-popover-ios.entry.js:2
n @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ ion-popover-ios.entry.js:313
(anonymous) @ index-2cd7e630.js:1581
Promise.then (async)
value @ index-2cd7e630.js:1581
(anonymous) @ createOverlayComponent.tsx:103
p @ 52.151ce889.chunk.js:2
(anonymous) @ 52.151ce889.chunk.js:2
(anonymous) @ 52.151ce889.chunk.js:2
n @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (async)
n @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ createOverlayComponent.tsx:104
(anonymous) @ createOverlayComponent.tsx:73
p @ 52.151ce889.chunk.js:2
(anonymous) @ 52.151ce889.chunk.js:2
(anonymous) @ 52.151ce889.chunk.js:2
n @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ createOverlayComponent.tsx:78
os @ react-dom.production.min.js:212
du @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
ji @ react-dom.production.min.js:122
mu @ react-dom.production.min.js:248
Xs @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
ji @ react-dom.production.min.js:122
Ki @ react-dom.production.min.js:123
Wi @ react-dom.production.min.js:122
re @ react-dom.production.min.js:287
Jt @ react-dom.production.min.js:68
```

### Expected Behavior

Popover item works and no error is generated.
See Firefox for proper behavior.

### Steps to Reproduce

1. Code a popover element
```
<React.Fragment>
                <IonText
                  className="works-link ion-hide-lg-down"
                  onClick={(e: any) => {
                    e.persist();
                    setShowExplorePopover({ showPopover: true, event: e });
                  }}>
                  Explore
                </IonText>
                <ExplorePopover
                  showExplorePopover={showExplorePopover}
                  setShowExplorePopover={setShowExplorePopover}
                />
              </React.Fragment>
```
2. View page element with Chromium build 114.0.5735.90 or 114.0.5735.91
3. See console for error

### Ionic Info

did not produce output?

**Comment by thetaPC** [comment]
Closing since this is a duplicate of [issue #27581](https://github.com/ionic-team/ionic-framework/issues/27581).

TL;DR:
The issue does not occur on the latest version of Ionic and it's recommended to upgrade. We do not [support updates](https://ionicframework.com/docs/reference/support) to v4 or v5.

There's a [migration guide](https://ionicframework.com/docs/updating/6-0) from Ionic v5 to v6 and a [migration guide](https://ionicframework.com/docs/updating/7-0) v6 to v7 that will help migrating easier.
